master/rock.py

```diff
@@ -84,13 +84,13 @@ def getPodSpec(self, build):
         spec = pod_def['spec']
 
         cpu = build.getProperty('parallel_build_level', 1)
-        memory = build.getProperty('memory_per_build_process_G', 2)
+        memory_k = int(build.getProperty('memory_per_build_process_G', 1.5) * 1024)
```

Comment:
Any reason for reducing this?

Reply:
Increasing the parallel build level per core.